packages/app-runtime/src/multiAccount/MultiAccountController.ts

```diff
@@ -238,6 +251,22 @@ export class MultiAccountController {
         await this._localAccounts.update(oldAccount, renamedAccount);
     }
 
+    public async updateLocalAccountDeletionDate(address: string, deletionDate?: CoreDate): Promise<void> {
+        const oldAccount = await this._localAccounts.findOne({ address });
+
+        if (!oldAccount) {
+            throw TransportCoreErrors.general.recordNotFound(LocalAccount, address).logWith(this._log);
+        }
+
+        const account = LocalAccount.from(oldAccount);
+
+        account.deletionDate = deletionDate ?? undefined;
+        await this._localAccounts.update(oldAccount, account);
+
+        const cachedAccount = this.sessionStorage.findSession(address)?.account;
```

Comment:
@jkoenig134 Should this always be defined? (I would like to return the updated cachedAccount, ideally knowing that it isn't undefined.)

Reply:
I think you could run "findOrCreateSession" IDK if it's called like this to make sure that it is defined. But I'd be careful about this cache and not rely on it too much.